### PR TITLE
doc: update README

### LINKED ISSUE #4: Plugin Documentation
@Oghenebrume50 @AlbertoPdRF I'm currently having a blocker setting up oh-my-zsh locally.

I'll be willing to write documentation for the plugin if i'm not able to successfully complete setup before deadline.

I'm making progress with setup though, I successfully upgraded my windows to enable me setup WSL (Windows Subsystem for Linux)

### PR DESCRIPTION
**Pull request by OPARA-PROSPER**
Fixes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
...
+- [Jq](https://stedolan.github.io/jq/) - jq is a lightweight and flexible command-line JSON processor. To install run `$ sudo apt install jq` on linux or `brew install jq` on OS X
+- Working **zsh** and **oh-my-zsh installation**
+
+### Testing Locally
+
+This plugin is not part of the default plugins that comes with the official Oh My Zsh shell and this means you'll need to
+setup locally to be able to use.
+
+- Fork and clone this repository locally to your machine
+- Move the directory to your **.oh-my-zsh** plugin directory in `~/.oh-my-zsh/plugin`
```
Comment (Oghenebrume50) [inline README.md:19]: this should be ~/.oh-my-zsh/custom/plugin

File: README.md [hunk 2]
```diff
@@ -1,2 +1,25 @@
...
+- Open your zshrc file `$ nano ~/.zshrc` and add the plugin `plugin=(git 0.0.1-oh-my-dev)`
```
Comment (Oghenebrume50) [inline README.md:20]: this should be ```plugin=(oh_my_dev)```

**Comment by Oghenebrume50** [comment]
Yeah you are right @AlbertoPdRF doing that now

**Comment by AlbertoPdRF** [comment]
Is the posting functionality missing?

**Comment by Oghenebrume50** [comment]
> Is the posting functionality missing?

No, it is there in the master branch

**Comment by AlbertoPdRF** [comment]
Would you mind adding it so we can merge this?

**Comment by Oghenebrume50** [comment]
Yes I will do that.

**Comment by Oghenebrume50** [comment]
Yes sure doing that now

**Pull request merged**